vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/dao/VpnServiceStateStatsDao.kt

```diff
@@ -19,16 +19,23 @@ package com.duckduckgo.mobile.android.vpn.dao
 import androidx.room.*
 import com.duckduckgo.mobile.android.vpn.model.BucketizedVpnServiceStateStats
 import com.duckduckgo.mobile.android.vpn.model.VpnServiceStateStats
+import kotlinx.coroutines.flow.Flow
 
 @Dao
 interface VpnServiceStateStatsDao {
 
-    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insert(stat: VpnServiceStateStats)
 
     @Query("SELECT strftime('%Y-%m-%d', timestamp) day, * FROM vpn_service_state_stats WHERE timestamp >= :startTime order by timestamp DESC")
     fun getServiceStateStatsSince(startTime: String): List<BucketizedVpnServiceStateStats>
 
+    @Query("SELECT * FROM vpn_service_state_stats ORDER BY timestamp DESC limit 1")
+    fun getLastStateStats(): VpnServiceStateStats
```

Comment:
This should also be nullable in case the db is empty
```suggestion
    fun getLastStateStats(): VpnServiceStateStats?
```